artichoke-backend/src/extn/core/symbol/symbol.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class Symbol
+  include Comparable
+
+  def capitalize
+    (to_s.capitalize! || self).to_sym
```

Comment:
some of these implementations from mruby are a bit funky. For example, the docs for [`Symbol#capitalize`](https://ruby-doc.org/core-2.6.3/Symbol.html#method-i-capitalize) say the method is equivalent to:

```ruby
sym.to_s.capitalize.intern
```

I'd like to match the docs (even if the semantics are the same) if possible.